### PR TITLE
rename signature files from zigbert to mozilla (bug 1152102)

### PR DESCRIPTION
**Pull request by magopian**
Fixes [bug 1152102](https://bugzilla.mozilla.org/show_bug.cgi?id=1152102)

### CODE REVIEW

File: apps/addons/management/commands/sign_addons.py [hunk 1]
```diff
@@ -27,12 +27,12 @@ def handle(self, *args, **options):
                 'Please provide at least one addon id to sign. If you want to '
                 'sign them all, use the "process_addons --task sign_addons" '
                 'management command.')
-        signing_server = options.get('signing_server', settings.SIGNING_SERVER)
-        preliminary_signing_server = options.get(
-            'preliminary_signing_server', settings.PRELIMINARY_SIGNING_SERVER)
+        full_server = options.get('signing_server') or settings.SIGNING_SERVER
+        prelim_server = (options.get('preliminary_signing_server') or
```
Comment (diox) [inline apps/addons/management/commands/sign_addons.py]: No tests ?

Comment (magopian) [inline apps/addons/management/commands/sign_addons.py]: added `apps/addons/tests/test_commands.py`

File: apps/addons/tests/test_commands.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    assert not settings.SIGNING_SERVER
+    assert not settings.PRELIMINARY_SIGNING_SERVER
+
+    # No endpoint defined in the test settings.
+    def no_endpoint(ids):
+        assert not settings.SIGNING_SERVER
+    mock_sign_addons.side_effect = no_endpoint
+    call_command('sign_addons', 123)
+    assert mock_sign_addons.called
+
```
Comment (robhudson) [inline apps/addons/tests/test_commands.py]: Do you need a `reset_mock()` in between here?

**Comment by robhudson** [comment]
r+wc

File: apps/addons/tests/test_commands.py (hunk 2 above)
Comment (magopian) [inline apps/addons/tests/test_commands.py]: Yes indeed, very well spotted, thanks!

**Pull request merged**